DEPENDENCY-NOTES.md

```diff
@@ -0,0 +1,8 @@
+As of 2024 August 22:
```

Comment:
I just realized this on SUSHI, but we forgot to update the date when we last updated this list. Since Git will track when this file gets updated, I sort of feel like this date isn't necessary and (clearly) is likely to get out of date. I'd personally vote to not list the date here. What do others think?

Reply:
We decided to keep the date here.